src/screens/ComponentLibraryScreen.tsx

```diff
@@ -37,26 +37,29 @@ function ComponentLibraryScreen() {
       <PurpleBackdrop>
         <Button variant='secondary' label='Secondary Button Outline' type='outline' />
       </PurpleBackdrop>
+      <PurpleBackdrop>
+        <Button variant='secondary' label='Secondary Button without style' />
+      </PurpleBackdrop>
 
-      <Text className='w-full mt-6 mb-6 font-bold text-center'>Headers</Text>
+      <Text className='w-1/2 mt-6 mb-6 font-bold text-center'>Headers</Text>
```

Comment:
Why has this reverted to `w-1/2`? I changed it to `w-full` so that the titles would be centred correctly.

Reply:
Yeah was a merge conflict i have sorted in my other branch, will do the same here.